scripts/run_locust.sh

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
```

Comment:
maybe rename this file to `run_quota_tests.sh`? We don't need to know its locust carrying out the tests.